app/controllers/concerns/content_items_concern.rb

```diff
@@ -0,0 +1,19 @@
+module ContentItemsConcern
```

Comment:
But then your suggestion seeks to associate/define these  concrete implementation i.e the methods withing the controller.. At what point would it be acceptable to introduce a data access or query object ??